carbon-design-system/carbon-for-ibm-dotcom
# fix(feature-section): add the correct token for the background container

### Related Ticket(s)

GH Issue: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/6571

### Description

- This PR changes the Background color of the section container to use `$ui-01` instead of `$hover-ui` token.

### Testing Instruction

- Visit the `Feature section` - `/?path=/story/components-feature-section--default`
- Ensure that the section container background color is set to `$ui-01`.

### Expected Display

![Screen_Shot_2021-07-30_at_2_36_13_PM](https://user-images.githubusercontent.com/1815714/127700617-633f7015-32a2-4320-a7aa-7a87ca71c96a.jpg)
